### PR TITLE
Version updates 0.24.1

### PR DESCRIPTION
**Pull request by tkatila**

### CODE REVIEW

**Review by mythi — APPROVED** [review]
looks great, thanks

File: charts/sgx-device-plugin/templates/sgx.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ apiVersion: deviceplugin.intel.com/v1
 kind: SgxDevicePlugin
 metadata:
   name: {{ .Values.sgx.metadata.name }}
+  annotations: {{ toYaml .Values.sgx.metadata.annotations | nindent 4 }}
```
Comment (mythi) [inline charts/sgx-device-plugin/templates/sgx.yaml:10]: don't we have to loop `range` here?

Comment (tkatila) [reply charts/sgx-device-plugin/templates/sgx.yaml:10]: It's not necessary. _.annotations_ is converted into an object by toYaml which is then indented appropriately.

Comment (mythi) [reply charts/sgx-device-plugin/templates/sgx.yaml:10]: thanks

**Review by mythi — APPROVED** [review]

**Comment by tkatila** [comment]
Rearranged the commits so that version update is at the top. Maybe not needed but just to be sure everything comes in for the new release.

**Review by poussa — APPROVED** [review]

**Pull request merged**